### PR TITLE
Add GrovePi drivers

### PR DESCRIPTION
**Pull request by aninternetof**
This is @rjfendricks and @ductape's work from Spring 2019 Hackday.

It adds the GrovePi drivers as a submodule, and adds then to the Dockerfile and setup.sh.

This also includes systemd setup.

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -13,20 +14,26 @@ if [ ! -e ${ADAFRUIT_DIR} ]; then
 	exit 2
 fi
 
+if [ ! -e ${GROVEPI_DIR} ]; then
+	echo "Error: GrovePi directory not found."
+	exit 2
```
Comment (cabarnes) [inline setup.sh]: This should probably be a different error code than the previous error

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +1,7 @@
 [submodule "Adafruit_Python_GPIO"]
 	path = Adafruit_Python_GPIO
 	url = https://github.com/aninternetof/Adafruit_Python_GPIO
+[submodule "GrovePi"]
+	path = GrovePi
+	url = https://github.com/rovercode/GrovePi.git 
+	branch = firmware-fixes-1.3.0
```
Comment (cabarnes) [inline .gitmodules]: This branch will always exist? Should we tag something here or merge it?

Comment (aninternetof) [reply .gitmodules]: I'll merge it.

File: setup.sh (hunk 1 above)
Comment (aninternetof) [reply setup.sh]: Fixing.

Comment (aninternetof) [reply setup.sh]: I didn't see any Unix error codes that were particularly meaningful for us, so I just incremented them so at least code inspection will point you to the right place.

File: .gitmodules (hunk 2 above)
Comment (aninternetof) [reply .gitmodules]: Done.

**Pull request merged**